consultation_analyser/consultations/export_user_theme.py

```diff
@@ -101,8 +107,20 @@ def get_theme_mapping_output(consultation: Consultation) -> list[dict]:
         # Default to latest execution run
         sentiment_run = get_latest_sentiment_execution_run_for_question_part(question_part)
         answer_qs = Answer.objects.filter(question_part=question_part)
+        # Get themes from latest framework
+        current_theme_mappings = ThemeMapping.get_latest_theme_mappings(
+            question_part=question_part, history=False
```

Comment:
Just to check: Does this mean we don't edit theme mappings, but only create or delete them?

Reply:
I think this will get current_theme_mappings, even when we edit them.

However, this does now raise a question of "should" we just create or delete thememappings and not edit them? Might be cleaner? That's what we do now in the eval app.

I'll make a ticket for us to think about how we do it.